src/containers/DonorList/index.js

```diff
@@ -53,6 +55,22 @@ export class DonorList extends Component {
                             <td>{donor.phone}</td>
                             <td>{donor.address}</td>
                             <td>Institution</td>
+                            <td>
+                                <Button
```

Comment:
Can we add icons instead of buttons with labels? We can add them later too once we're done with the essentials. 
https://www.w3schools.com/bootstrap/bootstrap_ref_comp_glyphs.asp

Reply:
sure, lets add them later, I just want to get functionality working for now